### DISCUSSION TITLE
能否增加个人自定义词库功能？

### DISCUSSION THREAD

**Discussion by LarryKHite** [immersive-translate/immersive-translate#970] [Ideas]
### 新功能 | New Feature

<img width="489" alt="20230930_121737" src="https://github.com/immersive-translate/immersive-translate/assets/9220139/5e035d75-88e2-440a-8da4-71b39279546b">

比如 “LLMs” 总是会被翻译成 “法学硕士”，如果能有一个自定义个人词库，我就可以设置 LLMs = 大型语言模型，这样可以大大提升翻译的准确性。

真的很希望能增加此功能，十分感谢！

### 为什么您想要添加这些功能 | Why you want to add these features

可以大大提升翻译的准确性

**Comment by Nokic233** [comment]
![image](https://github.com/user-attachments/assets/bea5c14d-1114-4cd4-8f7d-c38466b3bf2d)

我目前只知道可以这样，设置不翻译
